app-config.yaml

```diff
@@ -151,7 +151,7 @@ catalog:
       target: https://github.com/janus-idp/backstage-showcase/blob/main/catalog-entities/all.yaml
 
     - type: url
-      target: https://github.com/janus-idp/software-templates/blob/main/showcase-templates.yaml
+      target: https://github.com/redhat-developer/red-hat-developer-hub-software-templates/blob/main/templates.yaml
```

Comment:
shall we update this as well https://github.com/janus-idp/backstage-showcase/blob/main/.ibm/pipelines/resources/config_map/configmap-app-config-rhdh.yaml#L75

looks like used for testing 

cc @subhashkhileri

Reply:
Good catch @invincibleJai. It probably shoudl be updated.